2021-03-29 12:34:58,683 ERROR: Exception on /set_billing_settings [POST] [in /hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/hdd/code/eastbournecottage/app/routes.py", line 635, in set_billing_settings
    Payment_Breakpoint.due_by: datetime.strptime(row['dueBy'], '%Y-%m-%d').date(),
TypeError: strptime() argument 1 must be str, not None
2021-03-29 19:18:26,033 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:18:26,087 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:18:26,115 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:18:26,129 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:27:25,335 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:27:25,415 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:27:25,460 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:27:25,555 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:29:55,973 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:29:56,030 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:29:56,106 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:29:56,125 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:33:59,624 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:33:59,645 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:33:59,719 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:33:59,805 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:37:20,253 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:37:20,339 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:37:20,393 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:37:20,431 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:37:35,910 ERROR: Exception on /set_billing_settings [POST] [in /hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/hdd/code/eastbournecottage/app/routes.py", line 630, in set_billing_settings
    if row['deleteFlag']:
KeyError: 'deleteFlag'
2021-03-29 19:40:48,120 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:40:48,182 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:40:48,256 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:40:48,273 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:41:06,448 ERROR: Exception on /get_billing_settings [POST] [in /hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/hdd/code/eastbournecottage/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/hdd/code/eastbournecottage/app/routes.py", line 576, in get_billing_settings
    'dueBy': row.due_by.isoformat(),
AttributeError: 'int' object has no attribute 'isoformat'
2021-03-29 19:42:19,151 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:42:19,200 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:42:19,268 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 19:42:19,286 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,326 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,342 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,368 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,376 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,385 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,406 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,417 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-29 20:06:11,441 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:29:00,495 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:29:00,495 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:29:00,503 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:29:00,503 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:34:32,944 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:34:32,964 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:34:33,036 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
2021-03-30 14:34:33,092 INFO: eastbourne startup [in /hdd/code/eastbournecottage/app/__init__.py:29]
